assets/js/editor/index.tsx

```diff
@@ -43,13 +43,14 @@ export default {
     this.pushEventTo(this.el, this.changeEvent, { source: content });
   },
   render() {
-    const { adaptor, source } = this.el.dataset;
+    const { adaptor, source, disabled } = this.el.dataset;
```

Comment:
Please ensure you add `data-disabled` to the attrs list in the MutationObserver below.